cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,66 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPage from '../support/pages/settings.pageObject';
+import faker from 'faker';
+const settingsPage = new SettingsPage();
+const homePage = new HomePageObject();
+const signInPage = new SignInPageObject();
 
+const testData = {
+  email: faker.internet.email(),
+  username: faker.name.firstName().toLowerCase(),
+  password: faker.random.alphaNumeric(8),
+  bio: faker.lorem.lines(1)
+};
 describe('Settings page', () => {
+  let user;
   before(() => {
 
   });
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    cy.task('generateUser')
+      .then((generateUser) => {
+        user = generateUser;
+        cy.login(user.email, user.username, user.password);
+      });
+    settingsPage.visit();
   });
 
   it('should provide an ability to update username', () => {
-
+    settingsPage.clearUsernameField();
+    settingsPage.fillUsernameField(testData.username);
+    settingsPage.clickOnUpdateSettingsBtn();
+    homePage.assertHeaderContainUsername(testData.username);
+    cy.url().should('include', 'profile');
   });
 
   it('should provide an ability to update bio', () => {
-
+    settingsPage.fillUserBioField(testData.bio);
+    settingsPage.clickOnUpdateSettingsBtn();
+    homePage.assertUpdatedBio(testData.bio);
   });
 
   it('should provide an ability to update an email', () => {
```

Comment:
how do you assert here, that email has been changed? you just clicking update button, and check url